setup.py

```diff
@@ -14,7 +14,7 @@
     license='MIT',
     description='SendGrid Backend for Django',
     long_description=open('./README.rst').read(),
-    install_requires=["sendgrid >= 3.5, < 4"],
+    install_requires=["sendgrid >= 3.5, < 4.0.4"],
```

Comment:
This change implies that django-sendgrid is compatible with sendgrid 4. Is this actually the case? And if so, why don't we correct the entire version range?